lib/ARToolKitJS.cpp

```diff
@@ -313,9 +323,30 @@ extern "C" {
 		if ((arc->surfaceSet[surfaceSetCount] = ar2ReadSurfaceSet(datasetPathname, "fset", NULL)) == NULL ) {
 		    ARLOGe("Error reading data from %s.fset\n", datasetPathname);
 		}
-		ARLOGi("  Done.\n");
-
-	if (surfaceSetCount == PAGES_MAX) exit(-1);
+        // int surfaceSetCount = arc->surfaceSetCount;
+        int numIset = arc->surfaceSet[i]->surface[0].imageSet->num;
+        arc->nft.width_NFT =
+                    arc->surfaceSet[i]->surface[0].imageSet->scale[0]->xsize;
+        arc->nft.height_NFT =
+                    arc->surfaceSet[i]->surface[0].imageSet->scale[0]->ysize;
+        arc->nft.dpi_NFT =
+                    arc->surfaceSet[i]->surface[0].imageSet->scale[0]->dpi;
+
```

Comment:
comments need a `\n` at the end otherwise they will be printed on the same line. And that is very odd..